packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -473,7 +473,8 @@ export class List<T = any> extends React.Component<IListProps<T>, IListState<T>>
   }
 
   private _onRenderPage = (pageProps: IPageProps<T>, defaultRender?: IRenderFunction<IPageProps<T>>): any => {
-    const { onRenderCell, role } = this.props;
+    const { onRenderCell, role, items: allItems } = this.props;
+    const totalItemCount = allItems ? allItems.length : 0;
```

Comment:
I don't love accessing `props.items` here, but it doesn't feel like the value belongs on `pageProps` either. Since we're already accessing `props` it feels the least bad.